x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -8,7 +8,7 @@ export const eventsIndexPattern = 'logs-endpoint.events.*';
 export const alertsIndexPattern = 'logs-endpoint.alerts-*';
 export const metadataIndexPattern = 'metrics-endpoint.metadata-*';
 export const metadataCurrentIndexPattern = 'metrics-endpoint.metadata_current-*';
```

Comment:
As this will not follow the index pattern, this should change. I'm starting to wonder if we should even put the data under `metrics` prefix or outside? If we put it under `metrics-*` it can't contain to `-`.